src/js/components/PlacementConstraintsPartial.js

```diff
@@ -155,8 +138,24 @@ export default class PlacementSection extends Component {
               <option value="">Select</option>
               {this.getOperatorTypes()}
             </FieldSelect>
+            <FieldHelp>Specify where your app will run.</FieldHelp>
             <FieldError>{operatorError}</FieldError>
           </FormGroup>
+          <FormGroup
+            className={commonFieldsClassNames}
+            required={true}
+            showError={Boolean(fieldNameError)}
+          >
+            {fieldLabel}
+            <FieldInput
+              name={`constraints.${index}.fieldName`}
+              type="text"
+              placeholder="hostname"
+              value={constraint.fieldName}
+            />
+            <FieldHelp>Ex: zone, region, hostname</FieldHelp>
```

Comment:
I see you snuck in these help texts. 😉  Why didn't you also change the "Value" field help text to be like in the [prototypes](https://mesosphere.invisionapp.com/d/#/console/11374342/261629378/preview)?

Reply:
Mostly because I missed it 🤔 
See comment below for more details.